packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -165,6 +172,10 @@ export const Snackbar: FC<SnackbarProps> = (toast) => {
     },
   }
 
+  if (isAndroid()) {
+    containerProps.tabIndex = 0
+  }
+
```

Comment:
Had to move this out because in the app, running on iOS would get the following error:

`node_modules/@department-of-veterans-affairs/mobile-component-library/src/components/Snackbar/Snackbar.tsx(160,5): error TS2353: Object literal may only specify known properties, and 'tabIndex' does not exist in type 'ViewProps'.`

Reply:
Weird. I had noticed TS giving a red squiggly line when prototyping in the app node_modules on one ticket, but it seemed to run fine in iOS/Android previously. Oh well.